src/util/Constants.js

```diff
@@ -18,7 +18,8 @@ exports.DefaultOptions = {
     userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.67 Safari/537.36',
     ffmpegPath: 'ffmpeg',
     bypassCSP: false,
-    proxyAuthentication: undefined
+    proxyAuthentication: undefined,
+    cipherMsg: false
```

Comment:
Shouldn't default to true?

Reply:
This is off because the message can be decrypted. (Although this rarely happens, but I have encountered it a couple of times). And since they will have the same IDs, this will require extra processing on the developer side and needs to be understood and turned on by the developer himself.

Reply:
Can delete the parameter and send it to `message_ciphertext` without settings